quaireaux/data_structures/cairo_project.toml

```diff
@@ -0,0 +1,5 @@
+[crate_roots]
+data_structures = "src"
+utils = "../utils/src"
+
+tests = "tests"
```

Comment:
I think as for now I would rather go with the convention that exists in `core`: there is `#[cfg(test)]`ed `tests` module within package source

Reply:
Will update 😄